build_projects/dotnet-host-build/build.sh

```diff
@@ -93,8 +93,8 @@ done < "$REPOROOT/branchinfo.txt"
 [ -z "$DOTNET_INSTALL_DIR" ] && export DOTNET_INSTALL_DIR=$REPOROOT/.dotnet_stage0/$(uname)
 [ -d "$DOTNET_INSTALL_DIR" ] || mkdir -p $DOTNET_INSTALL_DIR
 
-
-$REPOROOT/scripts/obtain/dotnet-install.sh --channel preview
+DOTNET_INSTALL_SCRIPT_URL="https://raw.githubusercontent.com/dotnet/cli/rel/1.0.0/scripts/obtain/dotnet-install.sh"
+curl -sSL "$DOTNET_INSTALL_SCRIPT_URL" | bash /dev/stdin --verbose
```

Comment:
@brthor suggested using the default channel (which I believe is the RTM version you mentioned), which means not passing a 'channel' arg. This appears to have worked for non-windows, I'm trying the same for Windows.